styles/stylesheet1.css

```diff
@@ -0,0 +1,386 @@
+* {
+    box-sizing: border-box;
+  }
+
+h1, h2, h3, label, legend {
+  font-family: 'Cormorant Garamond', serif;
+}
+
+p, a {
+  font-family:  'Work Sans', serif;
+}
+
+header.head {
+    background-image: url(../images/header-image.jpg);
+    background-size: cover;
+    padding: 2em 4em 4em;
+    padding-bottom: 10em;
+    color: white;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.head-logo {
+  width: 33%;
+}
+
+header h1 {
+    color: #77ad3b;
+    font-style: italic;
+    font-weight: 500;
+    font-size: 175%;
+    margin-top: 0%;
+}
+
+.checkout-button, .checkout-button a {
+    float: right;
+    clear: both;
+    background-color: #77ad3b;
+    padding-top: .5em;
+    padding-bottom: .5em;
+    padding-left: 3em;
+    padding-right: 3em;
```

Comment:
You could combine these into

```
padding: .5em 3em;
```

It's also good practice to use leading 0's in front of values less than zero 

```
0.5rem
```